kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/MetadataColumnTestUtils.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.utils
+
+import io.delta.kernel.types.{MetadataColumn, StructField}
+import io.delta.kernel.types.StructField.createMetadataColumn
+
+trait MetadataColumnTestUtils {
+  val ROW_INDEX: StructField =
+    createMetadataColumn("_metadata.row_index", MetadataColumn.ROW_INDEX)
+  val ROW_ID: StructField = createMetadataColumn("_metadata.row_id", MetadataColumn.ROW_ID)
+  val ROW_COMMIT_VERSION: StructField =
+    createMetadataColumn("_metadata.row_commit_version", MetadataColumn.ROW_COMMIT_VERSION)
```

Comment:
I think I suggested adding static default fields for these - if we go ahead with that maybe we can remove this all-together? seems like a useful util for both the src code and test code